source/aggregation-tutorials/one-to-one-join.txt

```diff
@@ -20,69 +20,79 @@ One-to-One Join
-are then combined into one document.
+In this tutorial, you can learn how to use the {+driver-short+} to
+perform a one-to-one join. A one-to-one join occurs when a document
+in one collection has a field value that matches a single document in
+another collection with the same field value. The aggregation matches
+these documents on the field value and combines information from both
+into one document.
 
 .. tip::
 
-   A one-to-one join does not require the entities to have a :wikipedia:`One-to-one relationship
-   </One-to-one_(data_model)>`.
+   A one-to-one join does not require the documents to have a
+   One-to-one relationship. To learn more about this data relationship,
+   see the Wikipedia entry about :wikipedia:`One-to-one (data model)
+   <w/index.php?title=One-to-one_(data_model)&oldid=1096960092>`.
 
-The goal of this aggregation tutorial is to list all orders
-made in 2020, with the product information associated with each order.
+Aggregation Task Summary
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+This aggregation tutorial demonstrates how to join data from two
+collections related to product information and customer orders.
+The aggregated result is a list of all orders placed in 2020 that
+includes the product details associated with each order.
 
 This example uses two collections:
 
-- ``products`` which contains documents detailing a shop's available products.
-- ``orders`` which contains documents describing an individual order for a product at that shop.
+- ``orders``, which contains documents describing an order
+  for a product from a shop
+- ``products``, which contains documents that describe the available
+  products from a shop
 
-Because an order can be for only one product, the aggregation uses a
+An order can only contain one product, so the aggregation uses a
 one-to-one join to match an order document to the document for the ordered
-product. The collections are joined using a field called ``product_id`` that exists in
-the documents in both collections.
-
-The examples and sample data in this tutorial are sourced from the `Practical MongoDB
-Aggregations Book <https://www.practical-mongodb-aggregations.com/>`__.
+product. The collections are joined by a field called ``product_id``
+that exists in documents in both collections.
 
 Before You Get Started
 ----------------------
 
-Before you get started with this tutorial, you will need a Node.js application
-with the {+driver-long+} installed that connects to a MongoDB deployment.
+Before you start following this tutorial, set up the aggregation
+tutorial template app in file called ``agg_tutorial.js`` by following the
+instructions in the :ref:`node-agg-tutorial-template-app` section of the
+Aggregation Tutorials landing page.
 
-To learn how to create a MongoDB deployment on MongoDB Atlas, or
-on a local MongoDB instance, see the :ref:`Connection Guide <node-connect-to-mongodb>`.
+Get a handle on the ``products`` and ``orders`` collections:
```

Comment:
Maybe you're using standard phrasing I'm unfamiliar with, but I think this reads clearer:

```suggestion
Create handles for the ``products`` and ``orders`` collections:
```

Reply:
changed wording